README.md

```diff
@@ -1,31 +1,654 @@
+
+- A method named `purchaseTicket()` that takes no parameters and returns a `boolean`.
+    - If the number of available tickets is positive, decrement the number
+      of available tickets by 1 and return `true`.  Return `false` if there are
+      no tickets available.
+- A method named `addtoWaitList()` that takes no parameters and does not return a value.
+    - Increment the `waitlist` value by 1.
+
+Edit the `ConcertTest` Junit test class to add the following test methods.
+
+```java
+@Test
+void purchaseTicket() {
+    assertEquals(10, c1.getAvailable());
+    assertTrue(c1.purchaseTicket());
+    assertEquals(9, c1.getAvailable());
+    assertTrue(c1.purchaseTicket());
+    assertEquals(8, c1.getAvailable());
+
+    assertEquals(2, c2.getAvailable());
+    assertTrue(c2.purchaseTicket());
+    assertEquals(1, c2.getAvailable());
+    assertTrue(c2.purchaseTicket());
+    assertEquals(0, c2.getAvailable());
+    //no tickets left, remain at 0
+    assertFalse(c2.purchaseTicket());
+    assertEquals(0, c2.getAvailable());
+    }
+
+@Test
+void addToWaitlist() {
+    assertEquals(0, c1.getWaitlist());
+    c1.addToWaitlist();
+    assertEquals(1, c1.getWaitlist());
+    c1.addToWaitlist();
+    assertEquals(2, c1.getWaitlist());
+
+    assertEquals(0, c2.getWaitlist());
+    c2.addToWaitlist();
+    assertEquals(1, c2.getWaitlist());
+    c2.addToWaitlist();
+    assertEquals(2, c2.getWaitlist());
+    }
+```
+
+## Task #2 - `ConcertRepository` class
+
+The `ConcertRepository` class will use an array to
+encapsulate a collection of `Concert` class instances.
+The `ConcertRepository` class models a subset of
+functionality found in the `ArrayList`
+class that is part of the Java Collections Framework.
+
+
+(1) Edit the `ConcertRepository` class to add two instance variables:
+
+- An array named `concerts` that stores `Concert` class instances.
+- An int named `currentSize` that stores the number of `Concert` objects
+  that have been added to the array.  This value may differ from the array size,
+  which represents the maximum number of concerts that may be added to the array.
+
+
+(2) Create a constructor for the `ConcertRepository` that takes one parameter named
+`maxSize`.  The constructor should initialize the array size using the `maxSize` variable.
+Note that initially all elements in the array will be `null` since the array
+stores instances of the reference type  `Concert`.
+
+(3) Generate a getter method for the `currentSize` instance variable named `getCurrentSize`.
+
+(4) Create a method named `add` that takes one parameter and returns a `boolean`.
+The parameter type should be `Concert`.
+
+- The `currentSize` instance variable represents the number of concerts that have
+  been added to the array. Initially this value is 0.
+- If the array is not full (i.e. the current size is less than the array length),
+  add the parameter object into the array using the current size as the index.  Increment
```

Comment:
Just for clarity, maybe put currentSize in back ticks so they know you are talking about the variable: `currentSize`.

Reply:
yes, updated.